packages/app/src/cli/commands/env/show.ts

```diff
@@ -0,0 +1,27 @@
+import {appFlags} from '../../flags.js'
+import {AppInterface} from '../../models/app/app.js'
+import {load as loadApp} from '../../models/app/loader.js'
+import {showEnv} from '../../services/env/show.js'
+import {output, path, cli} from '@shopify/cli-kit'
+import Command from '@shopify/cli-kit/node/base-command'
+
+export default class EnvShow extends Command {
+  static description = 'Print the environment variables for your app and extensions'
+
+  static hidden = true
+
+  static flags = {
+    ...cli.globalFlags,
+    ...appFlags,
+  }
+
+  public async run(): Promise<void> {
+    const commandConfig = this.config
+    const {flags} = await this.parse(EnvShow)
+    const directory = flags.path ? path.resolve(flags.path) : process.cwd()
+    const envFile = path.resolve(directory)
+    const app: AppInterface = await loadApp(directory, 'report')
+    output.info(await showEnv(app, {config: commandConfig}))
+    if (app.errors) process.exit(2)
```

Comment:
Will the previous output work if there are errors? Also, instead of exiting, we usually call abort with the error message.

Reply:
@gonzaloriestra I investigated this more. I took inspiration from this commit https://github.com/Shopify/cli/commit/a0434ac2b68f00061dc32a24c52b14d001f691ae and I think we want to exit when showing `env` because the command might output stuff even if the app info is invalid. I'm happy to remove this line though if you think it's not worth it.

Reply:
I think it's fine in this case to do nothing if the app has errors, I feel like this is more suited for the `info` task. But it's good to know that we can throw `AbortSilent` for these cases.